src/libraries/System.Collections.Immutable/tests/Frozen/ComparerPickerTests.cs

```diff
@@ -10,9 +10,9 @@ public static class ComparerPickerTests
     {
         private static StringComparerBase NewPicker(string[] values, bool ignoreCase)
         {
-            var c = ComparerPicker.Pick(values, ignoreCase, out int minLen, out int maxLenDiff);
+            StringComparerBase c = ComparerPicker.Pick(values, ignoreCase, out int minLen, out int maxLenDiff);
```

Comment:
I assumed we weren't enforcing this policy in test projects. Good to see those changed as well though.